airbyte-ci/connectors/pipelines/pipelines/commands/airbyte_ci.py

```diff
@@ -64,6 +66,27 @@ def get_latest_version() -> str:
     raise Exception("Could not find version in pyproject.toml. Please ensure you are running from the root of the airbyte repo.")
 
 
+def get_airbyte_repo() -> git.Repo:
+    """Get the airbyte repo."""
+    return git.Repo(search_parent_directories=True)
```

Comment:
~~Can we throw if we aren't in a git repo at all?~~
That's done by the caller below 👍

Reply:
Should we validate that the git repo we are in is, in fact, `airbytehq/airbyte`?